src/plugins/hidsim/hid_thread_msg.h

```diff
@@ -0,0 +1,144 @@
+ * tamas.k.lengyel@gmail.com for further information.                      *
+ *                                                                         *
+ * If you have received a written license agreement or contract for        *
+ * Covered Software stating terms other than these, you may choose to use  *
+ * and redistribute Covered Software under those terms instead of these.   *
+ *                                                                         *
+ * Source is provided to this software because we believe users have a     *
+ * right to know exactly what a program is going to do before they run it. *
+ * This also allows you to audit the software for security holes.          *
+ *                                                                         *
+ * Source code also allows you to port DRAKVUF to new platforms, fix bugs, *
+ * and add new features.  You are highly encouraged to submit your changes *
+ * on https://github.com/tklengyel/drakvuf, or by other methods.           *
+ * By sending these changes, it is understood (unless you specify          *
+ * otherwise) that you are offering unlimited, non-exclusive right to      *
+ * reuse, modify, and relicense the code.  DRAKVUF will always be          *
+ * available Open Source, but this is important because the inability to   *
+ * relicense code has caused devastating problems for other Free Software  *
+ * projects (such as KDE and NASM).                                        *
+ * To specify special license conditions of your contributions, just say   *
+ * so when you send them.                                                  *
+ *                                                                         *
+ * This program is distributed in the hope that it will be useful, but     *
+ * WITHOUT ANY WARRANTY; without even the implied warranty of              *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the DRAKVUF   *
+ * license file for more details (it's in a COPYING file included with     *
+ * DRAKVUF, and also available from                                        *
+ * https://github.com/tklengyel/drakvuf/COPYING)                           *
+ *                                                                         *
+ * This file was created by Jan Gruber.                                    *
+ * It is distributed as part of DRAKVUF under the same license             *
+ ***************************************************************************/
+
+#ifndef HID_THREAD_MSG_H
+#define HID_THREAD_MSG_H
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <mutex>
+#include <cstring>
+
+std::mutex g_lock;
+
+struct h_thr_msg
+{
+    /* Signifying, whether an action has to be taken */
+    bool has_to_be_processed;
+
+    /* Actual resolution dependent screen coordinates */
+    uint16_t x;
+    uint16_t y;
+
+};
+
+static struct h_thr_msg g_msg;
+
+bool write_msg(struct h_thr_msg msg)
```

Comment:
I don't see the write happening anywhere...

Reply:
A leftover, which was accidentally included by squashing the commits, sorry! Removed.